contracts/Vault.vy

```diff
@@ -825,8 +830,7 @@ def _issueSharesForAmount(to: address, amount: uint256) -> uint256:
     if totalSupply > 0:
         # Mint amount of shares based on what the Vault is managing overall
         # NOTE: if sqrt(token.totalSupply()) > 1e39, this could potentially revert
-        freeFunds: uint256 = self._totalAssets() - self._calculateLockedProfit()
-        shares =  amount * totalSupply / freeFunds  # dev: no free funds
+        shares =  amount * totalSupply / self.freeFunds()  # dev: no free funds
```

Comment:
```suggestion
        shares =  amount * totalSupply / self._freeFunds()  # dev: no free funds
```

Reply:
Thanks for pointing this out made a change to the function name but not the call 🤦‍♂️